tests/scale/virt-api.go

```diff
@@ -55,6 +56,14 @@ var _ = Describe("[sig-compute] virt-api scaling", decorators.SigCompute, func()
 
 			numberOfNodes = len(nodes.Items)
 		}
+
+		onOCP, err := clusterutil.IsOnOpenShift(virtClient)
+		Expect(err).ToNot(HaveOccurred(), "failed to detect cluster type")
+
+		if !onOCP {
+			Skip("scaling of virt-api via the customizeComponents field isn't possible on OCP")
```

Comment:
I would really advise against runtime skips. Who knows when this logic will malfunction and we'll start seeing true negatives.

You are much better off adding a label to the test and having the owner of the infra declare if they're interested in picking up this test for their run. In this case if the owner only cares about OCP, they'll filter out this test unconditionally.

Reply:
done, could you have a look ?

Reply:
/lgtm
what's the motivation btw? do these tests fail? normally, if your'e running on ocp, you'd scale down any uber operator that may interfere beforehand. was this not the case?